modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find logs for Knative Serving components
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
```

Comment:
I would suggest adding the link to appropriate assembly the additional resources section of the assembly this module will build in since we can't link module to module.

Reply:
I'd probably just include the re-used module to save confusion? (example in https://github.com/openshift/openshift-docs/pull/16154/files#diff-9ef19605df4e8cbc8ccf698f7e10aadeR13 if you're interested)